css/css-cascade/important-prop-ref.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
```

Comment:
nit: I usually just don't add these tags (`<html>`, `<head>`), since they add noise to the rest of the test.